### PR TITLE
Added home page tests.

### PR DESCRIPTION
**Pull request by nibivi77**
### Description

The tests are basic. Making sure the components are displayed and rendered.

### What to test for/How to test
`npm run test:unit`

### CODE REVIEW

File: __tests__/pages/home.test.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  beforeEach(() => {
+    useRouter.mockImplementation(() => ({
+      pathname: '/',
+      asPath: '/',
+    }))
+  })
+  it('should render', () => {
+    render(<Home />)
+    const enLink = screen.getByText('English')
+    expect(enLink).toBeInTheDocument()
```
Comment (TishamIslam) [inline __tests__/pages/home.test.js:36]: Seems like there's some missing French language tests?

Comment (nibivi77) [reply __tests__/pages/home.test.js:36]: Its added.

File: components/molecules/ContactUs.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { ContactCard } from '../atoms/ContactCard'
 
 export const ContactUs = ({ mainTitle, contactList }) => {
   return (
-    <div className="layout-container">
+    <div className="layout-container" data-testid="contactUs">
```
Comment (DaceyTom2) [inline components/molecules/ContactUs.js:6]: Cool, I've never seen data-testid used before

**Review by DaceyTom2 — APPROVED** [review]
Looks good

**Pull request merged**